manywheel/build_cuda.sh

```diff
@@ -47,7 +47,7 @@ if [[ -n "$DESIRED_CUDA" ]]; then
     # There really has to be a better way to do this - eli
     # Possibly limiting builds to specific cuda versions be delimiting images would be a choice
     if [[ "$OS_NAME" == *"Ubuntu"* ]]; then
-        echo "Switching to CUDA version $desired_cuda"
+        echo "Switching to CUDA version $DESIRED_CUDA"
```

Comment:
Nit (is there a doc explaining when one is more preferable then other?
```suggestion
        echo "Switching to CUDA version ${DESIRED_CUDA}"
```